.circleci/config.yml

```diff
@@ -438,28 +438,24 @@ jobs:
                 command: ./project.rb bigquerytest
 
   ui-unit-test:
-    executor: workbench-executor
+    machine:
+      image: ubuntu-2204:2022.04.1
     steps:
       - checkout-code
       - halt-test-check:
           dir_names: "ui"
-      - run:
```

Comment:
why remove this?

Reply:
Let's chat about this tomorrow because it is causing scope creep. The Terra UI code requires node 18. The standard Circle machine has it, so it's an easy fix. Our own image does not. This was me doing an experiment to see what it would take to get the unit tests to run on the standard image, but that isn't feasible everywhere.